elastic/integrations
# [Cloud Security] add misconfiguration latest transform to Wiz integration

## Proposed commit message

Introducing the latest transform for Wiz Cloud Configuration Finding data to support CDR workflows as per [[Guide] Adapting data stream to Cloud Security in Kibana](https://docs.google.com/document/d/1govaQj_8LwUOLtk3zrcjCEvRgyKWRuC-TUihCrk04ao/edit#heading=h.poyt4ylkm7e5)



## Checklist


## Author's Checklist



## How to test this PR locally



1. build the package `elastic-package build`
2. run stack `elastic-package stack up -v -d`
3. Install Wiz integration, enable Cloud Configuration Finding data stream
4. check that the transform is created and started. It should be healthy after some time
5. check that Wiz's findings are available in the Findings page

## Related issues


- contributes to https://github.com/elastic/security-team/issues/9664 

## Screenshots
